lib/src/message_list/message_list_flagged.dart

```diff
@@ -44,101 +44,59 @@ import 'package:delta_chat_core/delta_chat_core.dart';
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:ox_coi/src/invite/invite_mixin.dart';
 import 'package:ox_coi/src/l10n/l.dart';
 import 'package:ox_coi/src/l10n/l10n.dart';
-import 'package:ox_coi/src/message/message_item.dart';
 import 'package:ox_coi/src/navigation/navigatable.dart';
 import 'package:ox_coi/src/navigation/navigation.dart';
-import 'package:ox_coi/src/ui/dimensions.dart';
 import 'package:ox_coi/src/widgets/dynamic_appbar.dart';
-import 'package:ox_coi/src/widgets/state_info.dart';
 
-import 'flagged_bloc.dart';
-import 'flagged_events_state.dart';
+import 'message_list.dart';
+import 'message_list_bloc.dart';
+import 'message_list_event_state.dart';
 
-class Flagged extends StatefulWidget {
+class MessageListFlagged extends StatefulWidget {
   final int chatId;
 
-  const Flagged({Key key, this.chatId = Chat.typeStarred}) : super(key: key);
+  const MessageListFlagged({Key key, this.chatId = Chat.typeStarred}) : super(key: key);
 
   @override
-  _FlaggedState createState() => _FlaggedState();
+  _MessageListFlaggedState createState() => _MessageListFlaggedState();
 }
 
-class _FlaggedState extends State<Flagged> {
-  Navigation _navigation = Navigation();
-  FlaggedBloc _flaggedBloc = FlaggedBloc();
+class _MessageListFlaggedState extends State<MessageListFlagged> with InviteMixin {
+  final _navigation = Navigation();
+  final _scrollController = ScrollController();
+  final _bloc = MessageListBloc();
```

Comment:
```suggestion
  final _bloc = MessageListBloc();
```
```suggestion
  final _messageListBloc = MessageListBloc();
```
I think _messageListBloc would be a better name. It is understandable for everyone :)